src/styles/foundation/filters.scss

```diff
@@ -0,0 +1,44 @@
+@import '../polaris-tokens/color-filters.color-map';
+
+///
+/// Color filter data
+///
+/// Shopify color filter palette, extended specifically for polaris-react.
+///
+/// @type map
+$color-filter-palette-data: $polaris-color-filters;
+
+/// Returns the filter list for a given color name and group.
+///
+/// @param {String} $hue - The color’s hue.
+/// @param {String} $value - The darkness/lightness of the color. Defaults to
+/// base.
+/// @return {List} The filter list.
+
+@function filter($hue, $value: base) {
```

Comment:
Did you write this or did you copy it from somewhere else?

Reply:
I copied the `color` function and modified it to use the filter map instead.